### PR TITLE
Handles both MultiDiGraph and DiGraph without up-conversion

### PR DESCRIPTION
**Pull request by jackboyla**
For https://github.com/aplbrain/grand-cypher/issues/54 and https://github.com/aplbrain/grand-cypher/issues/52

@j6k4m8 @davidmezzetti

### CODE REVIEW

File: grandcypher/__init__.py [hunk 1]
```diff
@@ -754,6 +761,15 @@ def returns(self, ignore_limit=False):
             for key, values in results.items()
             if self._alias2entity.get(key, key) in self._return_requests
         }
+        # HACK: convert to [None] if edge is None
```
Comment (j6k4m8) [inline grandcypher/__init__.py:764]: this is clever!!

**Review by j6k4m8 — COMMENTED** [review]
this is amazing, great work! I want to test this with grand-based graphs to make sure we're hitting the right apis so this doesn't accidentally break any currently working code but I will do that ASAP and merge!

**Comment by j6k4m8** [comment]
Perfect! also lolololol @ the branch name @jackboyla!

File: grandcypher/__init__.py (hunk 1 above)
Comment (jackboyla) [reply grandcypher/__init__.py:764]: this is only necessary for DiGraph input, so I’m sure there’s a cleaner way to do it🙂

**Pull request merged**